README.md

```diff
@@ -106,6 +106,35 @@ If you are looking for a tracker, here are some other projects worth noting:
 - [**AlphaPose**](https://github.com/MVIG-SJTU/AlphaPose) just released a new version of their human pose tracker. This tracker is tightly integrated into their code base, and to the task of tracking human poses.
 - [**SORT**](https://github.com/abewley/sort) and [**Deep SORT**](https://github.com/nwojke/deep_sort) are similar to this repo in that they use Kalman filters (and a deep embedding for Deep SORT), but they are hardcoded to a fixed distance function and to tracking boxes. Norfair also adds some filtering when matching tracked objects with detections, and changes the Hungarian Algorithm for its own distance minimizer. Both these repos are also released under the GPL license, which might be an issue for some individuals or companies because the source code of derivative works needs to be published.
 
+## Benchmarks
+Using our demo [motmetrics4norfair](https://github.com/tryolabs/norfair/tree/master/demos/motmetrics4norfair), we evaluated a Norfair tracker on the [MOT17](https://motchallenge.net/data/MOT17/) challenge. 
```

Comment:
Simpler is better, also mention this is not meant for production? Something like:

"MOT17 results obtained using [motmetrics4norfair](https://github.com/tryolabs/norfair/tree/master/demos/motmetrics4norfair) demo script. Hyperparameters where tuned for reaching a high MOTA on this dataset. A more balanced set of hyperparemters, like the default ones used in the other demos, is recommended for production."

Reply:
Ok

Reply:
Just in case, mind the typo:

> Hyperparameters ~where~ **were** tuned for reaching